src/Build/BackEnd/Components/Communications/NodeProviderOutOfProcBase.cs

```diff
@@ -121,7 +122,17 @@ protected void ShutdownAllNodes(long hostHandshake, long clientHandshake, NodeCo
             // For all processes in the list, send signal to terminate if able to connect
             foreach (Process nodeProcess in nodeProcesses)
             {
-                Stream nodeStream = TryConnectToProcess(nodeProcess.Id, 30/*verified to miss nodes if smaller*/, hostHandshake, clientHandshake);
+                // Some nodes can take up to this long to respond, so this is the smallest it can be without erroneously missing nodes.
```

Comment:
This is what the original comment meant, but let’s add context to the magic number. I believe this was a 2013 comment that didn’t provide methodology. Best to say that.

Reply:
I imagine they just created a lot of processes, then ran a build to see if it would pick up the extant nodes, but I wasn't there in 2013. What was the methodology?

Reply:
You don't have to describe the methodology, just don't state it as fact now. Otherwise, future me might email future you in five years demanding an explanation of how you knew this. And you won't know!

The internal commit that introduced this comment was http://vstfdevdiv:8080/DevDiv2/DevDiv/_versionControl/changeset/865139?_a=compare&path=%24%2FDevDiv%2Ffeature%2FVSPro_VBCS%2Fvsproject%2Fxmake%2FXMakeBuildEngine%2FBackEnd%2FComponents%2FCommunications%2FNodeProviderOutOfProcBase.cs